graphql_compiler/compiler/ir_lowering_common/common.py

```diff
@@ -70,10 +75,12 @@ def to_match(self):
         return mark_name
 
 
-def lower_context_field_existence(ir_blocks, query_metadata_table):
+def lower_context_field_existence(
+    ir_blocks: List[BasicBlockT], query_metadata_table: QueryMetadataTable
+) -> List[BasicBlockT]:
     """Lower ContextFieldExistence expressions into lower-level expressions."""
 
-    def regular_visitor_fn(expression):
+    def regular_visitor_fn(expression: Any) -> Any:
```

Comment:
Would narrowing this from `Any` to `Expression` work? It seems like the only place that uses this is `visit_and_update_expressions` which takes a `visitor_fn: Callable[[Expression], Expression]`

(similar logic might apply to other functions type-hinted as `Any`->`Any`)

Reply:
I'm not sure why it's showing up as Any now, I was pretty sure I changed this locally and pushed the changes 🤔 ... I'll watch out for this next time I push to this PR.